apps/namadillo/src/App/Common/GasFeeModal.tsx

```diff
@@ -283,15 +282,26 @@ export const GasFeeModal = ({
                         </div>
                       </div>
                       <div className="text-right">
-                        {totalInDollars && (
-                          <FiatCurrency amount={totalInDollars} />
-                        )}
-                        <div className="text-neutral-500 text-xs">
-                          <TokenCurrency
-                            amount={displayAmount}
-                            symbol={asset.symbol}
+                        {isShielded && !shieldedAmount.data ?
```

Comment:
As we have this condition applied on two places, do you mind creating a variable like `isLoading = isShielded && !shieldedAmount?.data`? It'll help to be more intuitive when reading the code :)